scipy/special/_add_newdocs.py

```diff
@@ -2306,8 +2306,8 @@ def add_newdoc(name, doc):
 
             sn(u|m), cn(u|m), dn(u|m)
 
-        The value `ph` is such that if `u = ellipkinc(ph, m)`,
-        then `sn(u|m) = sin(ph)` and `cn(u|m) = cos(ph)`.
+        The value `ph` is such that if ``u = ellipkinc(ph, m)``,
```

Comment:
```suggestion
        The value ``ph`` is such that if ``u = ellipkinc(ph, m)``,
```

Reply:
do you know if numpydoc says anything about this? Reading https://numpydoc.readthedocs.io/en/latest/format.html#returns, I would probably opt for the single backticks, but it isn't clear

Reply:
You're right looked at this a bit too hastily